Pocket/pocket-ios
# Core Data - Refactor entities to match required and non-nullable fields with the graph

## Summary
* This PR:
    - Sets the code gen to `Manual/none` for all Core Data entities
    - Matches the nullability and uniqueness of fields with the objects in the graph

## Implementation Details
* All entities were updated as described in the summary.
> **Note**
since we are updating fields, Core Data will attempt a lightweight migration. This might cause instability and/or crashes while updating from a previous build. It is recommended to delete and reinstall. This issue will not affect production users migrating from Pocket 7 since they won't have a previous version of the Core Data DB.

## Test Steps
* Build/run this branch and make sure that the app works as expected. Especially look for crashes or any data inconsistency in Home and Saves

## PR Checklist:

## Screenshots
